x-pack/legacy/plugins/file_upload/public/legacy.ts

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { npStart } from 'ui/new_platform';
+import { plugin } from '.';
+
+const pluginInstance = plugin();
+
+export const start = pluginInstance.start(npStart.core);
```

Comment:
Why the rename when it is being renamed yet again when it is imported?

Reply:
Related to the previous comment on naming it "start". [Here's the relevant section](https://github.com/elastic/kibana/blob/master/src/core/MIGRATION.md#3-export-your-runtime-contract) in the migration guide.